paparazzi-gradle-plugin/src/test/projects/verify-resources-kotlin/settings.gradle

```diff
@@ -3,3 +3,5 @@ apply from: '../test.settings.gradle'
 include ':consumer'
 include ':module1'
 include ':module2'
+
+enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
```

Comment:
can we move this to `../test.settings.gradle` and see if that can avoid adding to all these individual settings.gradle files?